Makefile

```diff
@@ -146,10 +146,10 @@ test-e2e: ##@tests Run e2e tests
 	# order: reliability then alphabetical
 	# TODO(tiabc): make a single command out of them adding `-p 1` flag.
 	build/env.sh go test -timeout 5m ./e2e/accounts/...
-	build/env.sh go test -timeout 5m ./e2e/api/...
-	build/env.sh go test -timeout 5m ./e2e/node/...
+	build/env.sh go test -timeout 5m ./e2e/api/... -network=3
```

Comment:
Haven't they been splitted due to different timeouts?

Reply:
1. Ah, I missed one thing! How is it supposed to be run with a specific networkid from CI?
2. Don't forget to use `-p 1` flag in case you decide to merge it. I'm also all for it but don't do it if you face some weird behavior.
3. Why are you using `network=3` there? `-network` should be specified from CI for all tests with no exception.

Reply:
I considered skipping those tests that require a valid network URI when `statuschain` is supplied as a option. Am not sure, if we want to group everything together.

Reply:
No, initially all were `40m`. I split them and ordered when we started resolving flakiness. We can keep it like it is for some time but still I don't get why we have `-network` only in some of them.

Reply:
> Don't forget to use -p 1 flag in case you decide to merge it. I'm also all for it but don't do it if you face some weird behavior.

@tiabc What is `-p 1` flag? (I remember I already asked about it but I don't remember in which PR...)

Reply:
@influx6 sorry, didn't get your answer. There're 3 questions and I see an answer only for the third.
3. Do you have any arguments for grouping or not grouping?

Reply:
`-p 1` makes `go test` run tests in different packages run in sequence, not in parallel.
In our case it makes the app throw errors about concurrent datadir usage as we use the same datadir in every test.

Reply:
Thinking abit more, it really won't matter either we if combined or split apart. So feel free to decide.

@tiabc Just saw your reply, the reason was at the moment to deal with test that needed network URLs, but having added skipping for such test, these will be removed.

My attention is focused to ensure all test keep passing, as before.

Reply:
> How is it supposed to be run with a specific networkid from CI?

In CI we should run all tests with the same settings, right? However, shouldn't it depend on the context? For instance, a regular PR is run only with `StatusChain` network but a PR to a release branch must run against all networks.

Reply:
@influx6 I thought @adambabik has already suggested and I supported:
>2. Don't forget to use -p 1 flag in case you decide to merge it. I'm also all for it but don't do it if you face some weird behavior.
? Please, try.

Reply:
It best to keep things separate for the main time. It stalls for sometimes during execution. We can revisit this at a later time on a later PR.